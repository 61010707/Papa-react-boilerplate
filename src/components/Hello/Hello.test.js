import React from "react";
import { shallow, mount, render } from "enzyme";

import Hello from "./index";

describe("Hello", () => {
  it("should render ", () => {
    const component = shallow(<Hello />);
    expect(component).toMatchSnapshot();
  });

  it("should mount", () => {
    const component = mount(<Hello />);
    expect(component).toMatchSnapshot();
  });

  it("should render", () => {
    const component = render(<Hello />);
    expect(component).toMatchSnapshot();
  });
});
