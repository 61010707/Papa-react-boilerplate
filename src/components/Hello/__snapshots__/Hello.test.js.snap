// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello should mount 1`] = `
<_default>
  <div>
    This is div
    <styled.div>
      <StyledComponent
        forwardedComponent={
          Object {
            "$$typeof": Symbol(react.forward_ref),
            "attrs": Array [],
            "componentStyle": ComponentStyle {
              "componentId": "sc-bdVaJa",
              "isStatic": true,
              "lastClassName": "jtOXgR",
              "rules": Array [
                "
  background-color: teal;
",
              ],
            },
            "displayName": "styled.div",
            "foldedComponentIds": Array [],
            "render": [Function],
            "styledComponentId": "sc-bdVaJa",
            "target": "div",
            "toString": [Function],
            "warnTooManyClasses": [Function],
            "withComponent": [Function],
          }
        }
        forwardedRef={null}
      >
        <div
          className="sc-bdVaJa jtOXgR"
        >
          This is Div with styled
        </div>
      </StyledComponent>
    </styled.div>
    <Styled(styled.div)>
      <StyledComponent
        forwardedComponent={
          Object {
            "$$typeof": Symbol(react.forward_ref),
            "attrs": Array [],
            "componentStyle": ComponentStyle {
              "componentId": "sc-bwzfXH",
              "isStatic": true,
              "lastClassName": "ugvnr",
              "rules": Array [
                "
  background-color: teal;
",
                "
  background-color: red;
  font-size: 30px;
",
              ],
            },
            "displayName": "Styled(styled.div)",
            "foldedComponentIds": Array [
              "sc-bdVaJa",
            ],
            "render": [Function],
            "styledComponentId": "sc-bwzfXH",
            "target": "div",
            "toString": [Function],
            "warnTooManyClasses": [Function],
            "withComponent": [Function],
          }
        }
        forwardedRef={null}
      >
        <div
          className="sc-bdVaJa sc-bwzfXH ugvnr"
        >
          This is Extra Div styled
        </div>
      </StyledComponent>
    </Styled(styled.div)>
  </div>
</_default>
`;

exports[`Hello should render  1`] = `
<div>
  This is div
  <styled.div>
    This is Div with styled
  </styled.div>
  <Styled(styled.div)>
    This is Extra Div styled
  </Styled(styled.div)>
</div>
`;

exports[`Hello should render 1`] = `
<div>
  This is div
  <div
    class="sc-bdVaJa jtOXgR"
  >
    This is Div with styled
  </div>
  <div
    class="sc-bdVaJa sc-bwzfXH ugvnr"
  >
    This is Extra Div styled
  </div>
</div>
`;
